src/plots/linePlot.ts

```diff
@@ -21,6 +22,28 @@ export module Plots {
       this.attr("stroke-width", "2px");
     }
 
+    /**
+     * Gets the interpolation function associated with the plot.
+     *
+     * @return string | (points: Array<[number, number]>) => string)
+     */
+    public interpolator(): string | ((points: Array<[number, number]>) => string);
+    /**
+     * Sets the interpolation function associated with the plot.
+     *
+     * @param interpolator Interpolation function
```

Comment:
Wrong param type here (missing `string`), and the TSDoc is incorrect. It should be something like this:
```
@param {string | points: Array<[number, number]>) => string} interpolator
```
Is there a way to list the valid `string` inputs?